userspace/libsinsp/runc.cpp

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright (C) 2013-2019 Draios Inc dba Sysdig.
+
+This file is part of sysdig.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+#include "runc.h"
+
+#include <cstring>
+
+#include "sinsp.h"
+#include "sinsp_int.h"
+
+namespace {
+
+const size_t CONTAINER_ID_LENGTH = 64;
+const size_t REPORTED_CONTAINER_ID_LENGTH = 12;
```

Comment:
maybe a note what these values represent?

Reply:
I'm not sure what to add that wouldn't duplicate the constant names or the static_assert below.

Reply:
what does "reported" mean here? is it what's reported to us? what we're reporting to someone else?